src/main/java/it/gov/pagopa/pu/sil/service/legacyauth/SilLegacyBasicAuthService.java

```diff
@@ -0,0 +1,37 @@
+package it.gov.pagopa.pu.sil.service.legacyauth;
+
+import it.gov.pagopa.pu.auth.dto.generated.AccessToken;
+import it.gov.pagopa.pu.organization.dto.generated.SilServiceLegacyBasicAuthConfig;
+import it.gov.pagopa.pu.sil.connector.actualization.LegacyBasicAuthService;
+import it.gov.pagopa.actualization.legacy.dto.generated.Credentials;
+import it.gov.pagopa.actualization.legacy.dto.generated.Token;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SilLegacyBasicAuthService {
+  private final Integer expirationTimeInSeconds;
+  private final LegacyBasicAuthService legacyBasicAuthService;
+
+  public SilLegacyBasicAuthService(
+    @Value("rest.auth.expiration-timeout-seconds.default") Integer expirationTimeInSeconds,
```

Comment:
this property is not meaningful... this seems to be a p4pa-auth config
```suggestion
    @Value("legacy-auth.default-expiration-seconds") Integer expirationTimeInSeconds,
```